### DISCUSSION TITLE
Question: Adjustments to Rainbow on MUXTestApp Flyout Pages

### DISCUSSION THREAD

**Discussion by mdtauk** [microsoft/microsoft-ui-xaml#7468] [Q&A]
This is just a quick Question/Proposal for the Rainbow LinearGradientBrush used for the CommandBarFlyout and MenuFlyout TestUI pages in the MUXTestApp.

@llongley  Would switching to this rainbow gradient, be a better test for these pages, so the blurring effect is more noticeable with the Acrylic material?  You added these with this commit and PR https://github.com/microsoft/microsoft-ui-xaml/pull/5179/
```Xaml
            <LinearGradientBrush StartPoint="0,0" EndPoint="1,1">
                <GradientStop Color="Red" Offset="0.000" />
                <GradientStop Color="Red" Offset="0.166" />
                <GradientStop Color="DarkOrange" Offset="0.167" />
                <GradientStop Color="DarkOrange" Offset="0.332" />
                <GradientStop Color="Goldenrod" Offset="0.333" />
                <GradientStop Color="Goldenrod" Offset="0.499" />
                <GradientStop Color="SeaGreen" Offset="0.500" />
                <GradientStop Color="SeaGreen" Offset="0.666" />
                <GradientStop Color="MediumBlue" Offset="0.667" />
                <GradientStop Color="MediumBlue" Offset="0.832" />
                <GradientStop Color="Indigo" Offset="0.833" />
                <GradientStop Color="Indigo" Offset="1.000" />
            </LinearGradientBrush>
```

![image](https://user-images.githubusercontent.com/7389110/122496208-45c50e00-cfe3-11eb-9092-8580e72e203c.png)
![image](https://user-images.githubusercontent.com/7389110/122496321-7442e900-cfe3-11eb-8811-f511aa272eaf.png)
![image](https://user-images.githubusercontent.com/7389110/122496454-b704c100-cfe3-11eb-8b19-c487c1f96398.png)

**Comment by StephenLPeters** [comment]
I like this, could you submit a PR?

**Comment by mdtauk** [comment]
> I like this, could you submit a PR?

Ok... I hope I have done this right.
https://github.com/microsoft/microsoft-ui-xaml/pull/5219
